server/main.lua

```diff
@@ -80,10 +68,23 @@ local function buildWhereClause(filters)
                 placeholders[#placeholders+1] = filters.states[i]
                 statePlaceholders[i] = 'state = ?'
             end
-            query = query .. string.format(' AND (%s)', table.concat(statePlaceholders, ' OR '))
```

Comment:
Don't we still need the parentheses around the OR statement to isolate it from the AND?

Reply:
It would be good to verify this, but 90% no

Reply:
As I tested, it worked well

Reply:
In this case the statement starts with or
`Where status=1 or status=2 and x=3 and y=4`

Reply:
the AND would still take precedence